src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -104,4 +98,11 @@ final class MockTestStore: Store, AppConfigCaching, PrivacyPreservingProviding,
 	var keySubmissionMetadata: KeySubmissionMetadata?
 	var testResultMetadata: TestResultMetadata?
 	var exposureWindowsMetadata: ExposureWindowsMetadata?
+	
+	// MARK: - EventRegistrationCaching
+
+	var checkinInfoScreenShown: Bool = false
+	var traceLocationsInfoScreenShown: Bool = false
+	var shouldAddCheckinToContactDiaryByDefault = true
```

Comment:
Just a style-comment: either declare the type everywhere or only if no default value is given. I prefer the latter.
```suggestion
	var checkinInfoScreenShown = false
	var traceLocationsInfoScreenShown = false
	var shouldAddCheckinToContactDiaryByDefault = true
```